File: Service_eHospital_DongNai_A/Implement/BenhAn_PhauThuatTrongMiengService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class BenhAn_PhauThuatTrongMiengService : BaseService<BenhAn_PhauThuatTrongMieng, IBenhAn_PhauThuatTrongMiengRepository>
    , IBenhAn_PhauThuatTrongMiengService
    {
    private readonly IBenhAn_PhauThuatTrongMiengRepository _BenhAn_PhauThuatTrongMiengRepository;
    public BenhAn_PhauThuatTrongMiengService(IBenhAn_PhauThuatTrongMiengRepository BenhAn_PhauThuatTrongMiengRepository) : base(BenhAn_PhauThuatTrongMiengRepository)
    {
    _BenhAn_PhauThuatTrongMiengRepository = BenhAn_PhauThuatTrongMiengRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A/BenhAn_PhauThuatTrongMieng.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { BenhAn_PhauThuatTrongMieng } from './BenhAn_PhauThuatTrongMieng.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class BenhAn_PhauThuatTrongMiengService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'BenhAn_PhauThuatTrongMieng_Id', 'BenhAnNgoaiTru_Id', 'TSB_Mau', 'TSB_TimMach', 'TSB_TieuDuong', 'TSB_BenhKhac', 'TienLuong_Id', 'CKM_TenNguoiThan', 'CKM_DiaChi', 'CKM_DienThoai', 'CKM_NgayCamKet'];

    List: BenhAn_PhauThuatTrongMieng[] | undefined;
    ListFilter: BenhAn_PhauThuatTrongMieng[] | undefined;
    FormData!: BenhAn_PhauThuatTrongMieng;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "BenhAn_PhauThuatTrongMieng";
    }
}

